packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -362,6 +396,10 @@ export default {
             };
         },
 
+        shouldShowFulfilmentSelector () {
+            return this.serviceType !== CHECKOUT_METHOD_DINEIN || this.availableFulfilment.times.length > 1;
```

Comment:
As part of this is the logical inverse of `isCheckoutMethodDineIn`, I'd be tempted to use that here:
```suggestion
            return !this.isCheckoutMethodDineIn || this.availableFulfilment.times.length > 1;
```